docs/developer-resources/guides/first-app-v3/realtime-updates-ionic-deploy.md

```diff
@@ -17,7 +17,7 @@ $ ionic cordova plugin add cordova-plugin-ionic@latest --save
 
 There are two unique values to provide: your app id and channel name. Sign into Appflow, then find the App Id on your app’s dashboard:
 
-![app id location](/img/guides/first-app-v3/app-id-location.png)
+![Screenshot highlighting the location of the App ID in the Appflow dashboard.](/img/guides/first-app-v3/app-id-location.png 'Appflow App ID Location')
```

Comment:
This one also needs to updated to exclude "screenshot". The same goes for the same sentences found in other files.